### ISSUE TITLE
Tagging in code blocks.

### ISSUE THREAD

**Issue by IAmAnubhavSaini** [roman-r-m/joplin-inline-tags-plugin#7]
Some programming languages use `#` for a comment, namely bash and python. In code blocks that start with \```LANG and end with ````, tag creation and tagging should not be done; at the least, introduce a toggle to disallow/allow tagging in code blocks.

When we type `#` in a code block, a tags container pops up. After cancelling via `esc` it goes away, but this is just extra step. The suggestion box, and tagging behavior also goes away with space.

- Example of tagging in code block

![image](https://user-images.githubusercontent.com/1573771/142745011-fd20d82e-4a76-4535-8996-2b8ebd7dd9d7.png)

## Information

```
uname -a      
Linux kali 5.14.0-kali4-amd64 #1 SMP Debian 5.14.16-1kali1 (2021-11-05) x86_64 GNU/Linux
```

![image](https://user-images.githubusercontent.com/1573771/142744993-1f09ba3a-4b23-4cfa-adca-4c929432d547.png)

```
Joplin 2.5.12 (prod, linux)

Client ID: c65acfb824cf482f82a0447d00c6404b
Sync Version: 3
Profile Version: 39
Keychain Supported: No

Revision: 884b86f
```

**Comment by NeerajGartia21** [comment]
can I work on this issue @roman-r-m

**Comment by roman-r-m** [comment]
> can I work on this issue @roman-r-m

Hi, is this related to GSoC?

**Comment by NeerajGartia21** [comment]
yeah partially. For GSoC proposal I'm working on the Email Plugin idea.
But  I want to contribute for this issue.
I think adding a toggle button would be better solution for this issue.

**Comment by NeerajGartia21** [comment]
@roman-r-m is there any private chat-box to discuss about gsoc proposals?

**Comment by roman-r-m** [comment]
> can I work on this issue @roman-r-m

ok, go for it

> @roman-r-m is there any private chat-box to discuss about gsoc proposals?

You can use discord or private messages on the forum. But at this stage you should focus on fixing an issue or 2,

**Comment by NeerajGartia21** [comment]
Yeah sure I'm working on this issue.
But along with I have to work on the proposal so I have some queries that I want to discuss with you.
How can I get the discord handlers?
because your profile on the forum is hidden so I'm not able to contact you.

**Comment by roman-r-m** [comment]
the link to discord in on the Joplin GSoC page
on discord i'm using the same name as here

**Comment by NeerajGartia21** [comment]
Ok. Thanks for the information @roman-r-m

**Comment by NeerajGartia21** [comment]
@roman-r-m I'm not able to find any discord server link.
can you please provide that.

**Comment by roman-r-m** [comment]
This has been fixed by @personalizedrefrigerator in #10
